db/schema.rb

```diff
@@ -11,11 +11,26 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150419054539) do
+ActiveRecord::Schema.define(version: 20150419175648) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "menu_items", force: :cascade do |t|
+    t.string   "name"
+    t.decimal  "price"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "menu_items_orders", force: :cascade do |t|
+    t.integer "order_id"
+    t.integer "menu_item_id"
+  end
+
+  add_index "menu_items_orders", ["menu_item_id"], name: "index_menu_items_orders_on_menu_item_id", using: :btree
```

Comment:
Since we are adding indexes I would suggest testing for that in your model spec. Something in line of: 
``` 
it { is_expected.to have_db_index ['index_menu_items_orders_on_menu_item_id']}
``` 
The same goes for the other indexes you are adding.